### PR TITLE
Add enable_mesos_input_plugin config option

### PR DESCRIPTION
**Pull request by gracedo**
https://jira.mesosphere.com/browse/DCOS-47301
Adds the `enable_mesos_input_plugin` config option.

Terraform Core PR: https://github.com/dcos-terraform/terraform-template-dcos-core/pull/2
DCOS PR: https://github.com/dcos/dcos/pull/4222

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -143,37 +143,33 @@ module "dcos-infrastructure" {
 /////////////////////////////////////////
 
 module "dcos-install" {
-  source  = "dcos-terraform/dcos-install-remote-exec/null"
```
Comment (geekbass) [inline main.tf]: Please set this source back.

File: main.tf [hunk 2]
```diff
@@ -143,37 +143,33 @@ module "dcos-infrastructure" {
...
-  version = "~> 0.0.0"
```
Comment (geekbass) [inline main.tf]: Please set this version back.

Comment (geekbass) [reply main.tf]: oh! actually this should be 0.1.0 I believe. @fatz sanity check.

**Comment by bernadinm** [comment]
manually ran the test. Here is the output, good to merge:
```
$ terraform fmt
$ terraform validate --check-variables=false
$ docker run -ti -v$(pwd):/a --entrypoint /usr/bin/terraform-docs fatz/terraform-jenkins md /a | tr -d '\r' > README.md
$ git status
On branch gracedo/enable_mesos_input_plugin_config_option
Your branch is up to date with 'origin/gracedo/enable_mesos_input_plugin_config_option'.
```

**Pull request merged**